peer.go

```diff
@@ -1205,10 +1188,6 @@ func (p *peer) handleError(msg *lnwire.Error) bool {
 	case msg.ChanID == lnwire.ConnectionWideID:
 		for chanID, chanStream := range p.activeMsgStreams {
```

Comment:
Seems like this `chanID` won't be used anymore.

Reply:
good catch, fixed!